src/Common/DTOs/Leaderboard/LeaderboardUserDto.cs

```diff
@@ -1,4 +1,5 @@
-﻿using SSW.Rewards.Domain.Entities;
+﻿using System.Text.RegularExpressions;
+using SSW.Rewards.Domain.Entities;
```

Comment:
This dependency shouldn't be here - this project should not have a reference to domain at all. I've added #1093 to address this